_drafts/2017-06-07-checkers.md

```diff
@@ -0,0 +1,115 @@
+---
+name: checkers
+layout: post_discourse
+title: Tackling the Research Compendium at #runconf17
```

Comment:
This post is the perfect way to lead off a few longer-form posts on unconf17 projects. Captures past work, pre-unconf discussions, complexity, and I love the Stone Soup section crediting team work and contributions from outside the core team.